teams/miri.toml

```diff
@@ -0,0 +1,14 @@
+name = "miri"
+subteam-of = "compiler"
+
+[people]
```

Comment:
do we want to sync this with a github team?

Reply:
Not sure. Do we?^^ What would that do for us?

Reply:
@RalfJung If it's synced with a team then you can give that team permissions to repos instead of individuals. So for example if there's a new member, once the PR adding them here is merged they will have the same permissions as every other member. Where as right now you would have to do that manually add them to each repository.

Reply:
That sounds sensible. But I also have no idea which permissions I have why.^^ I can edit issues both in rust-lang/rust and rust-lang/miri and rust-lang/unsafe-code-guidelines (and maybe more?).

Reply:
Seems fine as is.

Reply:
Well, the main thing is that if there is a distinct mir repository etc, we're trying to move away from granting permissions to individuals and move towards granting permissions to teams. So maybe if there are distinct miri repositories we might want to have a team for that. But it's not a big deal.

Reply:
I mean if you want to change https://github.com/rust-lang/miri/ to be entirely team-based in terms of permissions, I am fine with that. Let me know if there is anything I can/should do to make that happen.

Reply:
@RalfJung You can follow these instructions in #146 to add github syncing. It was written for teams but it works the exact same for subteams I believe.

Reply:
I'm no admin in the Miri repo, so I doubt I can set up any kind of syncing... also I don't think the relevant team even exists already on github so some org admin would probably have to create it?

Reply:
@RalfJung Yes sorry, the only thing you need to do is make the pull request adding github syncing, once that's merged someone else will update the miri permissions. I believe your team will be created once you've enabled it on GitHub.

Reply:
I don't understand what I can "enable on GitHub" without having any admin powers.

Reply:
I opened a PR at https://github.com/rust-lang/team/pull/281. But it's all very confusing, most of https://github.com/rust-lang/team/issues/146 and in particular everything in "Potential Questions" makes little sense in my situation (creating a new team, not adapting an existing one).